composer.json

```diff
@@ -11,6 +11,7 @@
             "email": "mike@mjphaynes.com"
         }
     ],
+    "minimum-stability": "beta",
```

Comment:
Why do we need a minimum stability setting here? I could not find any untagged/unstable releases used.

Reply:
removed